app/brave_generated_resources.grd

```diff
@@ -417,6 +417,7 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       </message>
     </messages>
     <includes>
+      <include name="IDR_BRAVE_GOOGLE_ANALYTICS_POLYFILL" file="resources/js/google_analytics_polyfill.js" type="BINDATA" />
```

Comment:
@bbondy do you think it's ok to put it in this directory or should we create a new one for things that aren't copied from chrome?

Reply:
I think it's fine